doc/node_misc.md

```diff
@@ -21,8 +21,8 @@ Definition:
 ```c++
 class AsyncResource {
  public:
-  AsyncResource(MaybeLocal<v8::Object> maybe_resource, v8::Local<v8::String> name);
-  AsyncResource(MaybeLocal<v8::Object> maybe_resource, const char* name);
+  AsyncResource(v8::Local<v8::Object> resource, v8::Local<v8::String> name);
+  AsyncResource(v8::Local<v8::Object> resource, const char* name);
```

Comment:
Programmer economics: make `resource` the last argument and give it a `= v8::Local<v8::Object>()` default value.

Good idea, bad idea?

Reply:
Good idea! Thanks. Done.